### DISCUSSION TITLE
Scala 3 support?

### DISCUSSION THREAD

**Discussion by albertpastrana** [laserdisc-io/fs2-aws#816] [Q&A] [answered]
Hi, first of all, thanks a lot for the library 🙌.

We are considering migrating some of our projects to Scala 3 and are reviewing the dependencies they use. I haven't been able to find any reference to supporting Scala 3 in the code, issues, pull requests or discussions and I was wondering if I've missed something or if it's just that it hasn't been discussed yet?

Any information about it will be welcome!

**Comment by semenodm** [comment, marked as answer]
i already started working to support scala 3. It's just the global events that distracted me from finishing the work.
also in scope of this work i might turn off the scala 2.12 support. i'm not sure about this but if the complexity of supporting this will be high, i will get rid of it

**Comment by semenodm** [comment]
#841
